src/main/java/de/tum/cit/aet/artemis/core/service/CourseService.java

```diff
@@ -542,6 +547,10 @@ private void deleteCompetenciesOfCourse(Course course) {
         competencyRepository.deleteAll(course.getCompetencies());
     }
 
+    private void deleteFaqOfCourse(Course course) {
```

Comment:
I'd suggest the name `deleteFaqsOfCourse` to keep it consistent

Reply:
integrated